EthOn_glossary.md

```diff
@@ -225,27 +225,27 @@ Merkle Patricia trees provide a cryptographically authenticated data structure t
 ### Message
 [`ethon:Msg`](http://ethon.consensys.net/Msg)   
 Data (as a set of bytes) and value (specified as Ether) that is passed between two Accounts, either through the deterministic operation of an autonomous object (Contract Account) or the cryptographically secure signature of an External Account.
+### Message call data
```

Comment:
This can also be part of a Tx that is not a CallTx. Not sure if we should specify this.